operations/upgrade-ert-use-artifactory.yml

```diff
@@ -0,0 +1,20 @@
+---
+- op: replace
+  path: /resources/name=elastic-runtime
+  value:
+    name: elastic-runtime
+    type: artifactory
+    source:
+      endpoint: {{artifactory_endpoint}}
+      repository: {{artifactory_repository}}
+      regex: 'cf-(?<version>.*)\.pivotal'
```

Comment:
It's extracting the version and putting it into a capture group named <version> as per the artifactory resource docs.